### ISSUE TITLE
Cannot generate swagger file

### ISSUE THREAD

**Issue by bryanjager** [lithnet/resourcemanagement-client#15]
Hello Ryan, this is a great solution it has helped us greatly.  I am encountering an issue.  We need to generate the swagger file, but I cannot seem to figure it out.  I tried https://localhost:port/api-docs/, https://localhost:port/api-docs/swagger.json and a few other combinations of v1/v2/resources/etc, but none seem to work.  

I checked out the SwaggerWCF project and tried to figure out how to generate the swagger.  It looks like all the swagger code is good, but it talks about an app.config file for self hosted, but I cannot seem to find one.  I was looking at: https://github.com/abelsilva/swaggerwcf.  Not sure if it is needed or not?

Sorry for the issue, but I am new to all of this.  I appreciate your assistance with generating the swagger file.

**Comment by ryannewington** [comment]
Hi @bryanjager 

I assume you are talking about the REST API project located at https://github.com/lithnet/resourcemanagement-webservice ?

I've spent the better part of this morning trying to figure this one out, and I'm not sure what is going on either. I've compared the code from when I first released the swagger support to the current version, and I can't see any changes that would have broken it. But it is definitely broken.

I'll need a bit more time with this one sorry. 

If you need something urgently, you can have a look at Peter Stapf's blog post where he has a swagger file for the API available for download

https://justidm.wordpress.com/2016/05/16/creating-mobile-apps-for-mim-with-powerapps-and-lithnet-restapi/

**Comment by bryanjager** [comment]
Yes, sorry. I was referring to the RESTful API. I am glad that you are having issues with it. Thank you for taking time to fix it. I will take a look at the link. Is that the latest version of the swagger file? It looks like it is only v1, not v2. I hope I can wait so we can use v2 or v1. 🙂

Thanks again!

**Comment by bryanjager** [comment]
Hello Ryan, any updates on this? If anything else, can you post the latest swagger file here? I appreciate your help on this.

**Comment by ryannewington** [comment]
@bryanjager sorry for the delay. I think I've found the problem. In the setup document, it says to use the ASP .NET v4.0 Classic app pool. This seems to break swagger. Try changing to the ASP .NET v4.0 app pool (make sure that its using integrated, not classic pipeline).

Let me know if that fixes it for you

**Comment by bryanjager** [comment]
Hello @ryannewington ,

Thank you for looking into this issue.  

I made the change as you described.  When trying to hit the server, we encountered an issue.  When viewing the server in a browser, we received the following error:

```

HTTP Error 500.24 - Internal Server Error
An ASP.NET setting has been detected that does not apply in Integrated managed pipeline mode.
Most likely causes:
•	system.web/identity@impersonate is set to true.
Things you can try:
•	If the application supports it, disable client impersonation.
•	If you are certain that it is OK to ignore this error, it can be disabled by setting system.webServer/validation@validateIntegratedModeConfiguration to false.
•	Move this application to an application pool using Classic .NET mode - for example, %SystemRoot%\system32\inetsrv\appcmd set app "Default Web Site/" /applicationPool:"Classic .NET AppPool"
(You can set "Classic .NET AppPool" to the name of another application pool running in Classic managed pipeline mode) 
```
I then did what the error asked for and changed the following node in web.config:

  &lt;system.web&gt;
    &lt;compilation targetFramework="4.5" /&gt;
    &lt;pages controlRenderingCompatibilityVersion="4.0" /&gt;
    &lt;identity impersonate="**false**" /&gt;
        &lt;authorization&gt;
            &lt;allow users="?" verbs="GET,POST" /&gt;
        &lt;/authorization&gt;

However, when we made that change, we received another error:
{"error": {
   "code": "resource-management-exception",
   "details": null,
   "message": "Failure Message: \nFailure Source: IdentityIsNotFound\nDetails: The requestor’s identity was not found.\nCorrelation ID:XXXXXXX",
   "target": "XXXXXXXXXXXXXXX"
}}

I assumed that this would not affect the operation of the server, but it looks like it affects the operation of the server.  I cannot do any operations on the server in integrated mode.

Is there a setting I am missing?

I appreciate your help on this issue.
